src/compiler/gcc.rs

```diff
@@ -438,6 +439,11 @@ where
     if !may_dist && !parsed_args.profile_generate {
         cmd.arg("-P");
     }
+    match kind {
+       CCompilerKind::Clang => { cmd.arg("-frewrite-includes"); },
+       CCompilerKind::GCC => { cmd.arg("-fdirectives-only"); },
+        _ => (),
+    };
```

Comment:
Not 100% sure this semi-color is useful.

Reply:
Removed it.